topsf/tasks/inference_v2/combine_base.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+
+import luigi
+import law
+import os
+
+from columnflow.tasks.framework.base import Requirements, CommandTask
+from columnflow.tasks.framework.mixins import (
+    CalibratorsMixin, SelectorStepsMixin, ProducersMixin, InferenceModelMixin,
+)
+from columnflow.tasks.framework.remote import RemoteWorkflow
+from topsf.tasks.inference import CreateDatacards
+from columnflow.util import dev_sandbox
+
+from topsf.tasks.inference_v2.fit_mixin import FitMixin
+
+
+class CombineBaseTask(
+    FitMixin,
+    CreateDatacards,
+    InferenceModelMixin,
+    ProducersMixin,
+    SelectorStepsMixin,
+    CalibratorsMixin,
+    law.LocalWorkflow,
+    RemoteWorkflow,
+    CommandTask,
+):
+    sandbox = dev_sandbox(law.config.get("analysis", "combine_sandbox"))
+
+    combine_verbosity = luigi.IntParameter(
+        default=2,
+        significant=False,
+        description="Verbosity of the combine tool",
+    )
+
+    combine_help = luigi.BoolParameter(
```

Comment:
Same as above, but maybe this could be `--print-help` to be more explicit and avoid clashing with the law's default `--help` flag.
```suggestion
    print_help = luigi.BoolParameter(
```